Vungle/src/main/java/com/mopub/mobileads/AdData.kt

```diff
@@ -0,0 +1,194 @@
+// Copyright 2018-2020 Twitter, Inc.
+// Licensed under the MoPub SDK License Agreement
+// http://www.mopub.com/legal/sdk-license-agreement/
+
+package com.mopub.mobileads
+
+import android.os.Parcelable
+import com.mopub.common.Constants
+import com.mopub.common.CreativeOrientation
+import kotlinx.android.parcel.Parcelize
+import java.util.TreeMap
+import kotlin.collections.HashMap
+
+@Parcelize
+data class AdData(
```

Comment:
@anton-blashchuk-zoolatech where are these Kotlin classes from, are they ported from some version of MoPub's SDK code

Reply:
I think this class not related to non-AndroidX patch issue. Can we remove it and Kotlin related gradle configration?

Reply:
done